types/testingutils/runner.go

```diff
@@ -221,8 +222,8 @@ var SSVDecidingMsgs = func(consensusData *types.ConsensusData, ks *TestKeySet, r
 }
 
 var DecidingMsgsForHeight = func(consensusData *types.ConsensusData, msgIdentifier []byte, height qbft.Height, keySet *TestKeySet) []*qbft.SignedMessage {
-	r, _ := consensusData.HashTreeRoot()
 	fullData, _ := consensusData.MarshalSSZ()
+	r := sha256.Sum256(fullData)
```

Comment:
should be 
byts, _ := consensusData.Encode()
r, _ := qbft.HashDataRoot(byts)
fullData, _ := consensusData.MarshalSSZ()

Reply:
@alonmuroch fixed